### DISCUSSION TITLE
Doesn't GitHub Support have any inquiries?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#33564] [Other Feature Feedback, Questions, & Ideas]
Doesn't GitHub Support have any inquiries?
I haven't received a reply for 2 days

**Comment by radar-blip** [comment]
Sometimes they are backed up due to the amount of questions they have to answer.  
And problems they have to solve. 👍🏻

**Reply by ghost** [reply]
Do you think I'll get an answer soon?

**Reply by radar-blip** [reply]
Of course! 😃 Can't tell you how soon though.

**Reply by ghost** [reply]
I can get it within this week, right?

**Reply by radar-blip** [reply]
I can't promise you that.

**Reply by ghost** [reply]
Do you happen to know when GitHub Support employees work?

**Reply by radar-blip** [reply]
They are probably located all over the world, so probably someone is working all the time.

**Reply by ghost** [reply]
Approximately how many days did it take when you asked the support?

**Reply by radar-blip** [reply]
I never asked for support 😆
